### PR TITLE
Update 02-set-up.Rmd

### PR DESCRIPTION
**Pull request by RandomAside**
Small typo corrections up to section 2.4.2 - also a comment about the flow of the first few parts.

A very interesting read by the way!

### CODE REVIEW

File: 02-set-up.Rmd [hunk 1]
```diff
@@ -115,12 +119,10 @@ process. This can be started in modern Windows versions by typing `Ctrl-Alt-Del`
...
-2. Start an activity monitor then execute the following code chunk.
-In it `lapply()` (or its parallel version `mclapply()`) is used to *apply* a function, `median()`, over every column in the data frame object
-`X` (see Section \@ref(the-apply-family) for more on the 'apply family' of 
-functions).[The
-reason this works is that a data frame is really a list of vectors, each vector forming a column.
-]
+2. Start an activity monitor then execute the following code chunk in it.
+`lapply()` (or its parallel version `mclapply()`) is used to *apply* a function, `median()`, over every column in the data frame object
+`X` (see Section \@ref(the-apply-family) for more on the 'apply family' of functions) (The reason this works is that a data frame is really a list of vectors, each vector forming a column).
+
```
Comment (Robinlovelace) [inline 02-set-up.Rmd]: No problem with shifting the footnote to the main body with brackets, but the punctuation in your suggestion is not right.

File: 02-set-up.Rmd [hunk 2]
```diff
@@ -141,7 +143,7 @@ How do the system output logs results on your system compare to those presented
 3. What do you notice regarding CPU usage, RAM and system time, during and after each of the three operations?
 4. Bonus question: how would the results change depending on operating system?
 
-```{r timings-tests, echo=FALSE, eval=FALSE}
+```{r timings-tests, echo=FALSE, eval=FALSE} <-- Is this intentionally not echoed? doesn't show in github so people wouldn't know it's there to review...unless they edit the doc...
```
Comment (Robinlovelace) [inline 02-set-up.Rmd]: Yes it's deliberately not shown.

File: 02-set-up.Rmd [hunk 3]
```diff
@@ -197,7 +199,7 @@ sudo apt-get install r-cran-rcurl # install the rcurl package
 ```
 
 
-Typical output from the second command is illustrate below:
+Typical output from the second command is illustrated below:
```
Comment (Robinlovelace) [inline 02-set-up.Rmd]: Well spotted.

File: 02-set-up.Rmd [hunk 4]
```diff
@@ -226,7 +228,7 @@ However, it is important to keep your R version relatively up-to-date, because:
 - Many R packages only work on recent versions on R.
 
 Release notes with details on each of these issues are hosted at [cran.r-project.org/src/base/NEWS](https://cran.r-project.org/src/base/NEWS).
-R release versions have 3 components corresponding to major.minor.patch changes.
+R release versions have 3 components corresponding to major, minor or patch changes.
```
Comment (Robinlovelace) [inline 02-set-up.Rmd]: This is the way R versions are formated, e.g. 3.3.1.

File: 02-set-up.Rmd [hunk 5]
```diff
@@ -287,7 +289,7 @@ On Windows the **installr** package helps manage and update R packages with syst
 For example the **Rtools** package for compiling C/C++ code on Window can be installed with the following command:
 
 ```{r, eval=FALSE}
-installr::install.rtools() 
+installr::install.rtools()  <-- should the R in Rtools be capitalised? (I am on a mac so can't check if case sensitive)....
```
Comment (Robinlovelace) [inline 02-set-up.Rmd]: No

File: 02-set-up.Rmd [hunk 6]
```diff
@@ -319,7 +321,7 @@ Next (after the exercises) we take a look at how to configure R using start-up f
 
 ## R startup
 
-Every time R starts a couple of files scripts are run default, as documented in `?Startup`.
+Every time R starts a couple of files scripts are run by default, as documented in `?Startup`.
```
Comment (Robinlovelace) [inline 02-set-up.Rmd]: Thanks.

**Comment by csgillespie** [comment]
Gone stale.

**Pull request closed without merging**